app/src/main/resources/io/apicurio/registry/storage/impl/sql/mssql.ddl

```diff
@@ -2,73 +2,85 @@
 -- DDL for the Apicurio Registry - Database: MS SQL Server
 -- *********************************************************************
 
-CREATE TABLE apicurio (prop_name VARCHAR(255) NOT NULL, prop_value VARCHAR(255));
+CREATE TABLE apicurio (prop_name NVARCHAR(255) NOT NULL, prop_value NVARCHAR(255));
```

Comment:
Why this change?

Reply:
NVARCHAR supports Unicode, so it's more general https://stackoverflow.com/questions/144283/what-is-the-difference-between-varchar-and-nvarchar